frontend/src/metabase/query_builder/components/notebook/NotebookDataPicker/NotebookDataPicker.tsx

```diff
@@ -57,6 +58,30 @@ export function NotebookDataPicker({
     [query, stageIndex, table],
   );
 
+  const pickerInfo = useMemo(
+    () => table && Lib.pickerInfo(query, table),
+    [query, table],
+  );
+
+  const getUrl = () => {
+    if (!pickerInfo || !tableInfo) {
+      return;
+    }
+
+    const { isModel, cardId, tableId, databaseId } = pickerInfo;
+
+    if (cardId) {
```

Comment:
I'd probably add some unit tests as well where `window.open` would be mocked.

Reply:
https://github.com/metabase/metabase/pull/46228/commits/9ea3cfc53150f4f9e2134082fc1b94b7448a20c5

Reply:
and https://github.com/metabase/metabase/pull/46228/commits/38eb48e7317d129c7a014b16ece3859cfdb88072